appintro/src/main/java/com/github/appintro/indicator/DotIndicatorController.kt

```diff
@@ -46,6 +46,7 @@ class DotIndicatorController(context: Context) : IndicatorController, LinearLayo
         this.slideCount = slideCount
 
         for (i in 0 until slideCount) {
+            if (slideCount == 1) { break }
```

Comment:
I would suggest you instead change this to (you need to move it down):

```suggestion
            if (slideCount == 1) { 
                dot.visibility = View.INVISIBLE
            }
```

This will make sure that the dot is still added to the hierarchy and it's hidden.
You can then amend the change to lines 71-73

Reply:
Okay I got the point, I will make the required changes.